utils/recordGamedb.ts

```diff
@@ -0,0 +1,47 @@
+import prisma from "~/utils/db";
+
+type GameStat = {
+  player_id: string;
```

Comment:
nit: в нашем проекте мы используем camelCase

```suggestion
  playerId: string;
```